Changelog.md

```diff
@@ -19,6 +19,7 @@ __New Features__
 - Adds window and skylight `GlassType` options of "low-e, high-solar-gain" and "low-e, low-solar-gain"; updates U-factor/SHGC lookup tables.
 - Updated water heater installation location defaulting to match ANSI 301-2022
 - Adds more error-checking for inappropriate inputs (e.g., HVAC SHR=0 or clothes washer IMEF=0).
+- Adds capability to model variable speed system power shedding controls in response to commands in the AHRI 1380 standard with maximum power ratio schedule
```

Comment:
How about making AHRI 1380 more of an example?
```suggestion
- Adds maximum power ratio detailed schedule for variable-speed HVAC systems (e.g., to model shedding controls per AHRI 1380).
```

Reply:
@jmaguire1 Is there an AHRI 1380 example?

Reply:
I'm not saying we need an AHRI 1380 example, just that AHRI 1380 itself is an example of what you can model with this new capability.